GLSP-1369: Update id locators in tests to align  with workflow example

tortmayr opened the pull request:
Part of https://github.com/eclipse-glsp/glsp/issues/1369

<!--
Thank you for your Pull Request. Please provide a description and review
the requirements below.

Contributors guide: https://github.com/eclipse-glsp/glsp/blob/master/CONTRIBUTING.md

Note: Security vulnerabilities should not be disclosed on GitHub, through a PR or any
other means. See [SECURITY.md](https://github.com/eclipse-glsp/glsp/blob/master/SECURITY.md),
to learn how to report vulnerabilities.
-->

#### What it does

- Update id locators in tests
- Switch to composite ts build
- Update to latest dev config
- Introduce `generate:index` helper script
- Regenerate indices
- Set version to 2.2.0 to align with other GLSP components
- Update & cleanup package dependencies

<!-- Include relevant issues and describe how they are addressed. -->


#### How to test

<!-- Explain how a reviewer can reproduce a bug, test new functionality or verify performance improvements. -->

#### Follow-ups

<!-- Please list potential follow-up work, including known issues, possible future work, identified technical debt, and potentially introduced technical debt. If the PR introduces technical debt, specify the reason why this is acceptable. Please create tickets and link them here. Please use the label "technical debt" for new issues when it applies. -->

#### Changelog

<!-- Please check, when if it applies to your change. -->

- [ ] This PR should be mentioned in the changelog
- [ ] This PR introduces a breaking change (if yes, provide more details below for the changelog and the migration guide)

haydar-metin commented on examples/workflow-test/scripts/repositories.ts:
```diff
@@ -90,7 +90,7 @@ function buildClient(options: GlobalOptions): void {
 
 function buildTheia(options: GlobalOptions): void {
-    exec('yarn', [], { cwd: repositoryFolder(options.folder, theiaRepository) });
+    exec('yarn && yarn browser build', [], { cwd: repositoryFolder(options.folder, theiaRepository) });
```
```suggestion
    if(exec('yarn', [], { cwd: repositoryFolder(options.folder, theiaRepository)})) {
        exec('yarn', ['browser', 'build'], { cwd: repositoryFolder(options.folder, theiaRepository) });
    }
```
Otherwise it fails on my machine.

haydar-metin replied:
Can you also fix in the command `theia-integration` the start part (line 204)

from

`exec('yarn', ['start:ws:debug'], { cwd: repositoryFolder(folder, theiaRepository) });`

to

`exec('yarn', ['browser', 'start:ws:debug'], { cwd: repositoryFolder(folder, theiaRepository) });`

haydar-metin reviewed (approved):
Looks great, unfortunately, I don't have the rights to update the PR for this reason I have a small request (see comment).

Thanks for the changes!

tortmayr commented:
@haydar-metin 
Thanks for the review. I have addressed your comments and also introduced GH action workflows to ensure that the playwright tests run on every RP and push to master
